src/tests/functional/plugin/cpu/shared_tests_instances/skip_tests_config.cpp

```diff
@@ -38,7 +38,7 @@ std::vector<std::string> disabledTestPatterns() {
         // TODO: Issue 43417 sporadic issue, looks like an issue in test, reproducible only on Windows platform
         R"(.*decomposition1_batch=5_hidden_size=10_input_size=30_.*tanh.relu.*_clip=0_linear_before_reset=1.*_targetDevice=CPU_.*)",
         // Skip platforms that do not support BF16 (i.e. sse, avx, avx2)
-        R"(.*(BF|bf)16.*(jit_avx(?!5)|jit_sse|ref).*)",
+        //R"(.*(BF|bf)16.*(jit_avx(?!5)|jit_sse|ref).*)",
```

Comment:
I guess this was a temporary change and should be reverted.

Reply:
done